ambry-store/src/main/java/com.github.ambry.store/StoreMetrics.java

```diff
@@ -25,36 +25,39 @@
  * Metrics for a specific store.
  */
 public class StoreMetrics {
```

Comment:
let us discuss these "tightening access specifier" changes that we make and have started following. `IntelliJ` may highlight things because it does not see a current use of a method or a variable outside the package, but it could very well be the case that these were authored to be accessed outside of the package. The idea of making some method of a package public only when you make a change in another package that needs it does not seem like a good practice.